app/models/ability.rb

```diff
@@ -0,0 +1,3 @@
+class Ability
+  include CanCan::Ability
+end
```

Comment:
- [ ] Kindly add permissions to your user so that he can be abble to delete a post if it is theirs or if they have an admin role
 #### Note: You can check thins link for more information about [cancan:ability](https://github.com/CanCanCommunity/cancancan/blob/develop/README.md#:~:text=Here%20follows%20an%20example%20of%20rules%20defined%20to%20read%20a%20Post%20model.) :+1: